account_credit_control/report/account_credit_control_analysis.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, tools
+
+
+class AccountCreditControlAnalysis(models.Model):
+    _name = "credit.control.analysis"
+    _description = "Credit Control Analysis"
+    _auto = False
+    _rec_name = "partner_id"
+
+    partner_id = fields.Many2one(
+        comodel_name="res.partner", string="Partner", readonly=True
```

Comment:
No need of string="...", since odoo automatically does the job.

```suggestion
        comodel_name="res.partner", readonly=True
```